test/integration/custom-routes/next.config.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
+  async rewrites () {
```

Comment:
There's no test that checks rewrite ability to copy params over to the destination.